lib/entities/movie.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+
+
+class Movie {
+  final String title;
+  final String originalTitle;
+  final genres = const [
+    'Animation',
+    'Family',
+    'Adventure',
+    'Fantasy',
+    'Comedy'
+  ];
+  final String poster;
+  final String backdrop;
+  final String releaseDate;
+  final String voteAverage;
+  final String overview;
+
+  const Movie(this.title, this.originalTitle, this.poster, this.backdrop,
+      this.releaseDate, this.voteAverage, this.overview);
+
+  List<Widget> getGenreElements() {
```

Comment:
You should not have UI-dependant code inside your entities. 
You can extract a Flutter widget per element, and do the mapping in another place.

Reply:
done